lib/panel.html

```diff
@@ -103,6 +103,15 @@
 					</div>
 
 					<div class="option">
+						<label class="option-label" for="option-interface">Test Interface</label>
+						<select class="option-input" id="option-interface">
+							<option value="object">Object</option>
+							<option value="tdd">TDD</option>
+							<option value="bdd">BDD</option>
```

Comment:
I considered it, but I haven't actually used it, and I wasn't sure if the this.remote convention worked within the qunit runner. If it operates the same as the others I'll add it to this PR if you'd like.